package.json

```diff
@@ -108,6 +108,7 @@
     "test": "jest --config=jest_config/jest.config.json --coverage",
     "pretest": "check-node-version --package",
     "dev": "node webpack_config/server.js",
+    "dev:https": "HTTPS=true node webpack_config/server.js",
```

Comment:
I was able to launch the HTTPS server with an outdated version of node. Does a command like `predev:https` need to be added?

Reply:
Good call, will do